setup.py

```diff
@@ -45,7 +45,7 @@
 
 logger = logging.getLogger(__name__)
 
-version = '2.3.4.post33'
+version = '2.3.4.post34'
```

Comment:
NIT For Airflow source code commits, because we do not have CI tests set up to catch simple bugs, we generally do integration tests. This is because we have sometimes found that we've made simple mistakes (forgetting to add an import), and then had to continually release new versions just to fix the bug we introduced (e.g. post23-post26 may all just be bugged versions).  

To work around that problem, we followed this process: 

1/ Commit the change (as you did here) and bump the version to a pre-release with a .dev# suffix (see [here](https://github.com/lyft/airflow/pull/311/files)) 
2/ Release the .dev# version via airflow-private. 
3/ Create an airflowinfra2 PR that bumps to the .dev# version. 
4/ Manually deploy the airflowinfra2 commit to Kyte staging and Airflow staging, and confirm nothing breaks / everything works as expected. That's the test. 
5/ Go back here, remove the .dev0 suffix, and repeat the release process for the .post# version without the .dev# suffix.
6/ Update the airflow version in airflowinfra2 to the non-dev0 version, and merge + deploy the airflowinfra2 PR. 

But again, if we set up CI tests in this repo (or you are confident way your change has no bugs), then of course it'd be simpler to just release without the .dev0. You can still do the integration tests as I described above before releasing the version bump to prod.